alacritty/src/display/mod.rs

```diff
@@ -570,6 +668,46 @@ impl Display {
             rects.push(visual_bell_rect);
         }
 
+        // Handle search and IME drawing/positioning.
+        match search_state.regex() {
```

Comment:
Is this for IME support in the searchbar?

Reply:
Yes, and we always had it.